client/main/infobox.js

```diff
@@ -28,39 +29,151 @@ class InfoBox extends HTMLElement {
             <div class="body">
                 <div class="local">
                     <slot>
-                        <h1>Hi,</h1>
-                        <div class="content">
-                        </div>
-                        <button>OK</button>
+                        <div class="heading">
+                            <svg fill="#eac282" height="30px" viewBox="0 0 24 24" width="30px" xmlns="http://www.w3.org/2000/svg">
```

Comment:
shift this into a chunk of css? (even if it's just a string defined somewhere in this file). it's nice to group all the css together.

Reply:
not sure i understand what you mean by moving it to css.

Reply:
for example:

```
.heading {
    height: 30px ;
    background-image: url('data:image/svg+xml;utf8,<svg ....
}
```

Reply:
yep, i didn't know you could do that but i guess it makes sense.